### ISSUE TITLE
Github Releases Modpacks

### ISSUE THREAD

**Issue by BobVarioa** [PrismLauncher/PrismLauncher#694]
### Role

Playing personal modpacks via PrismLauncher.

### Suggestion

An option to provide a github repository when you are importing a modpack. This would work in a similar way to the "Import from Zip" option, but instead ask for a repository url.

### Benefit

Playing a personal modpack with friends can get tiresome if you want to tweak the modpack; making sure all parties have their modpack up to date, making sure they have the right file to import, even getting them to import said file, etc.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by Scrumplex** [comment]
If you offer your modpack in any of the supported formats you can just paste the direct download url in the "import from zip" dialog

**Comment by BobVarioa** [comment]
Yes, but then that doesn't offer utilities like updating the modpack with just a press of a button, i.e. it's not a "first-class" solution.

**Comment by barraIhsan** [comment]
you can "kinda" do it by using auto updating pack using `packwiz`.
*btw, prism auto updating mods (check for updates) and downloading mods (.minecraft/mods/.index) is using packwiz*

**Comment by ghost** [comment]
Similar issue in polymc https://github.com/PolyMC/PolyMC/issues/424

**Comment by IceColdTray** [comment]
I assume mods are also included in this one?

**Comment by Syntoxr** [comment]
I'm also interested in this but would generalize this issue to target remote location modlists.

As mentioned above, Prism already supports custom links to remote locations but doesn't allow for saving them. 
Especially when using cloud providers for hosting modpack files, the URL can get quite "complex" and users wouldn't want to remember where they saved this URL in order to copy/paste it into Prism every time they want to update.

I suggest adding an input field as well as an `check for updates` button in order to perform the already implemented tasks automatically.

As a bonus, it would be nice to have a `notify on update` checkbox but as this also does not exist for mods, I'm unsure if this fits in the devs vision of this project.
